### PR TITLE
chore: reorganize some docs

### PR DESCRIPTION
**Pull request by bpmct**
some minor changes to docs structure

### CODE REVIEW

**Review by kylecarbs — APPROVED** [review]

File: docs/admin/configure.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-Coder server's primary configuration is done via environment variables. For a full list
-of the options, run `coder server --help` on the host.
+Coder server's primary configuration is done via environment variables. For a full list of the options, run `coder server --help` on the see our [CLI documentation](../cli/server.md).
```
Comment (matifali) [inline docs/admin/configure.md:1]:
```suggestion
Coder server's primary configuration is done via environment variables. For a full list of the options, run `coder server --help` or see our [CLI documentation](../cli/server.md).
```

**Review by matifali — APPROVED** [review]
You may want to run https://github.com/coder/coder/actions/workflows/cron-weekly.yaml after merging this.

File: docs/admin/configure.md (hunk 1 above)
Comment (matifali) [reply docs/admin/configure.md:1]: a minor typo I think.

**Review by matifali — CHANGES_REQUESTED** [review]
A minor typo I think.

**Pull request merged**